vendor/README.md

```diff
@@ -0,0 +1,31 @@
+# Third party packages
+
+This directory contatins packages from third-party vendors reused in Astar Network.
+
+### Why not fork?
+
+As a way to protect code from **unexpected changes** and **release from third-party project
+internal dependencies** a dedicated folder used instead of repository fork.
+
+In case of internal packages the porter persion will have no intention to just scroll down
+repository commit history and adjust final result to the repo. Copy/paste the code risk is
+still exist, but in this case all changes will be clearly viewed in PR created before merge.
+
+## Package list
+
+| Directory                          | Package name                   | Origin                                        |
+|------------------------------------|--------------------------------|-----------------------------------------------|
+| evm-tracing                        | moonbeam-client-evm-tracing    | ${moonbeam}/client/evm-tracing                | 
```

Comment:
How will `${moonbeam}` be interpreted in Github? Is it just raw text?

Reply:
Yes, it's raw text. We cant be sure is it github repo or local folder.